website/src/components/doc-page/doc-page.tsx

```diff
@@ -204,23 +205,37 @@ function useProductInformation(
     return null;
   }
 
-  const selectedName = result[1] || "";
+  const selectedPath = result[1] || "";
   const selectedVersion = result[2] || "";
   let stableVersion = "";
 
-  const selectedProduct = products?.find((p) => p?.path === selectedName);
+  const selectedProduct = products?.find((p) => p?.path === selectedPath);
 
   if (selectedProduct) {
     stableVersion = selectedProduct.latestStableVersion || "";
   }
 
   return {
-    name: selectedName,
+    path: selectedPath,
+    name: getProductNameFromPath(selectedPath),
     version: selectedVersion,
     stableVersion,
   };
 }
 
+function getProductNameFromPath(path: string): string | null {
```

Comment:
Why not taking the `title` instead of the `path`?

Reply:
What do you mean? This method is used to derive the name from the path segment and to later set the window title. So injecting the window title here wouldn't work, since we don't have it yet.

Reply:
My concern here is, that we would have to maintain the product names in two different places, whereas we could achieve to manage these names in one place instead by reading these from the JSON file.